### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by vivekjhauipath**

### CODE REVIEW

File: language_version_metadata/Python36_CV__2__metadata.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "processor": "CPU",
         "mlPackageLanguage": "PYTHON36_CV",
         "languageGroup": "CV",
-        "baseImage": "python36cv:v21.8.1-2",
+        "baseImage": "python36cv:v21.10.0-2",
```
Comment (snvenkat) [inline language_version_metadata/Python36_CV__2__metadata.json:7]: Make sure you update the SF with the right tag for CV.

**Review by snvenkat — APPROVED** [review]

File: metadata/AustralianInvoices__7__metadata.json [hunk 2]
```diff
@@ -20,7 +20,5 @@
     "minAIFabricVersion": "v21.10",
     "languageVersion": 2,
     "version": 7,
-    "baseImageCpu": "python37du:v21.8.1-2",
```
Comment (snvenkat) [inline metadata/AustralianInvoices__7__metadata.json:23]: Don;t we need this?

Comment (snvenkat) [reply metadata/AustralianInvoices__7__metadata.json:23]: baseImageCpu and baseImageGpu?

Comment (snvenkat) [reply metadata/AustralianInvoices__7__metadata.json:23]: How do we know which baseImage to use then?

Comment (snvenkat) [reply metadata/AustralianInvoices__7__metadata.json:23]: based on mlPackageLanguage?

File: language_version_metadata/Python36_CV__2__metadata.json (hunk 1 above)
Comment (vivekjhauipath) [reply language_version_metadata/Python36_CV__2__metadata.json:7]: yes, @mohitseth-nitrr is changing to these tags

File: metadata/AustralianInvoices__7__metadata.json (hunk 2 above)
Comment (vivekjhauipath) [reply metadata/AustralianInvoices__7__metadata.json:23]: already part of changelog, was here for testing. Will avoid few db calls if we remove

File: language_version_metadata/Python36_CV__2__metadata.json (hunk 1 above)
Comment (mohitseth-nitrr) [reply language_version_metadata/Python36_CV__2__metadata.json:7]: @snvenkat  builds are in progress and here is the PR for review https://github.com/UiPath/ai-pkgmanager/pull/594/files

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Pull request merged**